### PR TITLE
Fix quotes and remove extra commas in terraform template

### PR DESCRIPTION
**Pull request by jeremiahsnapp**
### :white_check_mark: Checklist

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [ ] Docs added/updated?
- [ ] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: terraform/test-environments/modules/chef_automate_install/templates/install_chef_automate_cli.sh.tpl [hunk 1]
```diff
@@ -129,23 +129,23 @@ configure_automate_infra_views() {
...
-            "admin_key": "$chef_server_admin_key",
-            "server_id": "$server_id",
-          }'
+      if ! chef-automate dev grpcurl automate-gateway -- chef.automate.api.infra_proxy.InfraProxy.GetOrg -d "{\"id\": \"$org_id\", \"server_id\": \"$server_id\"}" 2> /dev/null | grep "$org_id" &> /dev/null; then
+          chef-automate dev grpcurl automate-gateway -- chef.automate.api.infra_proxy.InfraProxy.CreateOrg -d "{
+            \"id\": \"$org_id\",
+            \"name\": \"${chef_server_org}\",
+            \"admin_user\": \"${chef_server_admin_name}\",
+            \"admin_key\": \"$chef_server_admin_key\",
+            \"server_id\": \"$server_id\",
```
Comment (vsingh-msys) [inline terraform/test-environments/modules/chef_automate_install/templates/install_chef_automate_cli.sh.tpl:147]: left one more comma

Comment (btm) [reply terraform/test-environments/modules/chef_automate_install/templates/install_chef_automate_cli.sh.tpl:147]: this is JSON, yeah? so no trailing commas, right?

Comment (vsingh-msys) [reply terraform/test-environments/modules/chef_automate_install/templates/install_chef_automate_cli.sh.tpl:147]: Yes, it is JSON, Golang struct made me culprit here :disappointed:

Comment (jeremiahsnapp) [reply terraform/test-environments/modules/chef_automate_install/templates/install_chef_automate_cli.sh.tpl]: thanks for catching that. the comma is gone now

**Pull request merged**